test/Hotel-test.js

```diff
@@ -0,0 +1,101 @@
+import chai from 'chai';
+import Hotel from '../src/Hotel'
+import rooms from '../src/data/Rooms-data'
+import Room from '../src/Room'
+import bookings from '../src/data/Bookings-data';
+import Booking from '../src/Booking'
+const expect = chai.expect;
+
+describe('Hotel', () => {
+let room1, room2, room3, room4, booking1, booking2, roomsData, bookingsData, newHotel;
+
+beforeEach(() => {
+  room1 = rooms[0]
+  room2 = rooms[1]
+  room3 = rooms[2]
+  room4 = rooms[3]
+  booking1 = bookings[0]
+  booking2 = bookings[1]
+  roomsData = [room1, room2, room3, room4]
+  bookingsData = [booking1, booking2]
+  newHotel = new Hotel(roomsData, bookingsData)
```

Comment:
Similarly to `customer repo test`, testing what if `roomsData` and `bookingsData` were `undefined` could be helpful to safeguard from unexpected outcomes.